src/api/app/models/event/delete_package.rb

```diff
@@ -4,6 +4,10 @@ class DeletePackage < Base
     self.description = 'Package deleted'
     payload_keys :project, :package, :sender, :comment, :requestid
 
+    def subject
+      "Package #{payload['package']} deleted"
```

Comment:
```suggestion
    "Package #{payload['project']}/#{payload['package']} deleted"
```
I would add the project part too, like in other places (build_unchanged, build_success, ...).